aggregator/list.go

```diff
@@ -354,18 +354,18 @@ func (l *metricList) discardLocalMetric(
 	l.metrics.flushMetricDiscarded.Inc(1)
 }
 
-// consumeForwardMetric consumes a forward metric.
+// consumeForwardedMetric consumes a forward metric.
 // TODO(xichen): implement this.
-func (l *metricList) consumeForwardMetric(
+func (l *metricList) consumeForwardedMetric(
 	metric aggregated.Metric,
 	meta metadata.ForwardMetadata,
 ) {
 
```

Comment:
nit: would it be worth panic'ing here just to ensure these functions aren't called until they are implemented?

Reply:
Sure can do, I'll add the implementation in the next PR but doesn't hurt either.